internal/api/graphql/graph/baseResolver/issue_match.go

```diff
@@ -120,6 +120,8 @@ func IssueMatchBaseResolver(app app.Heureka, ctx context.Context, filter *model.
 
```

Comment:
I assume line 97 is not required anymore?

```
       issue_match_ids := []*int64{}
	for _, issue_match_id := range filter.ID {
                filterById, err := ParseCursor(issue_match_id)
```

Reply:
Or maybe `ParseCursor` should be renamed?

Reply:
True